docs/configuration/index.md

```diff
@@ -422,6 +422,12 @@ There are several emitters available:
 
 ##### Logging Emitter Module
 
+The use this emitter module, set `druid.emitter=logging`. The Logging emitter will use the configured
+`druid.emitter.logging.loggerClass` to emit events. If the `loggerClass` is set to `LoggingEmitter` (the default), the
```

Comment:
> The Logging emitter will use the configured `druid.emitter.logging.loggerClass` to emit events. If the `loggerClass` is set to `LoggingEmitter` (the default), the events are logged as a single `json` object with a Marker…

All the `druid.emitter.logging.loggerClass` does is control the package name of the logger (what gets printed for `%c` in the pattern). The new doc line here makes it sound like it does more than that. Suggest something like:

> The `logging` emitter uses a log4j2 logger named `druid.emitter.logging.loggerClass` to emit events. Each event is logged as a single `json` object with a Marker…

Also, from looking at the code, I believe the existing doc is wrong: the default is `org.apache.druid.java.util.emitter.core.LoggingEmitter` not `LoggingEmitter`.